docs/methods/cardanoSignTransaction.md

```diff
@@ -1,46 +1,88 @@
+* `stakingKeyHash` - *optional* `string` Staking part of the address as hex string, either this or `stakingPath` are required for base address derivation, device cannot verify ownership of the staking key.
+* `certificatePointer` - *optional* `CardanoCertificatePointer` object. Must contain `number`s `blockIndex`, `txIndex` and `certificateIndex`. Required for pointer address derivation.
+
+#### CardanoCertificate
+* `type` - *required* `CardanoCertificateType` enum, currently supported are StakeRegistration (0), StakeDeregistration (1) and StakeDelegation (2)
+* `path` - *required* `string | Array<number>` BIP44 derivation path for the staking address included in this certificate, minimum length `5`, accepts both string (ex. `m/1852'/1815'/0'/2/0`) and Array (ex. `[(1852 | 0x80000000) >>> 0, (1815 | 0x80000000) >>> 0, (0 | 0x80000000) >>> 0, 2, 0]`) formats, must be a staking address (4th index must be a `2`)
+* `pool` - *optional* `string` Hash of pool ID in hexadecimal notation, required for certificates that represent a pool operation (currently StakeDelegation)
+
+#### CardanoWithdrawal
+* `path` - *required* `string | Array<number>` BIP44 derivation path for the reward address this withdrawal is for, minimum length `5`, accepts both string (ex. `m/1852'/1815'/0'/2/0`) and Array (ex. `[(1852 | 0x80000000) >>> 0, (1815 | 0x80000000) >>> 0, (0 | 0x80000000) >>> 0, 2, 0]`) formats, must be a reward address (4th index must be a `2`)
+* `amount` - *required* `number` The amount of ADA withdrawn to this address in Lovelace
+
+
+### Responses
+
+#### Success case
+```javascript
+{
+    success: true,
+    payload: {
+        hash: string, //blake2b hash of the transaction body, used to index this transaction in the blockchain
+        serializedTx: string, //cbor encoded signed transaction in hexadecimal notation
+    }
+}
 ```
 
-### Params 
-[****Optional common params****](commonParams.md)
-###### [flowtype](../../src/js/types/networks/cardano.js#L62-L109)
-* `inputs` — *obligatory* `Array` of [CardanoInput](../../src/js/types/networks/cardano.js#L71)
-* `outputs` - *obligatory* `Array` of [CardanoOutput](../../src/js/types/networks/cardano.js#L76)
-* `fee` - *obligatory* `String`
-* `ttl` - *obligatory* `String`
-* `protocolMagic` - *obligatory* `Integer` 764824073 for Mainnet, 42 for Testnet
-* `networkId` - *obligatory* `Integer` 1 for Mainnet, 0 for Testnet
-* `certificates` - *optional* `Array` of [CardanoCertificate](../../src/js/types/networks/cardano.js#L83)
-* `withdrawals` - *optional* `Array` of [CardanoWithdrawal](../../src/js/types/networks/cardano.js#L88)
-* `metadata` - *optional* `String`
+#### Error case
+```javascript
+{
+    success: false,
+    payload: {
+        error: string // error message
+    }
+}
+```
 
-### Stake pool registration certificate specifics
+### Internal validation details
```

Comment:
Why is this PR removing the original documentation for stake pool registration?

Reply:
a likely reason is that you want to add a separate PR for stake pool-related operations, but then I would rather do the removal there instead of this PR. In this state this isn't mergeable as it would break a pending PR for stake pool owner support in the official Trezor repo